projects/lf-documentation/src/app/getting-started/getting-started.component.html

```diff
@@ -1,10 +1,46 @@
 <p class="documentation-title">Getting Started</p>
-
-<p class="documentation-header">Using Framework-Agnostic Custom Elements</p>
+<p> UI components can be used via CDN script or NPM library. Below we list instructions on how to use the UI components.</p>
```

Comment:
UI components can be used via a CDN script or NPM package.

We don't need the below...